File: src/console/log/output/OutputManager.php
<?php

namespace pocketcloud\cloud\console\log\output;

final class OutputManager {

    private static ?OutputHandler $handler = null;

    public static function setHandler(OutputHandler $handler): void {
        self::$handler = $handler;
    }

    public static function getHandler(): OutputHandler {
        if (self::$handler === null) self::$handler = new DefaultOutputHandler();
        return self::$handler;
    }

    public static function reset(): void {
        self::$handler = new DefaultOutputHandler();
    }
}

File: src/main/java/de/pocketcloud/cloud/console/output/OutputManager.java
package de.pocketcloud.cloud.console.output;

import de.pocketcloud.cloud.console.output.impl.DefaultOutputHandler;

public final class OutputManager {

    private static OutputHandler outputHandler = null;

    public static void set(OutputHandler outputHandler) {
        OutputManager.outputHandler = outputHandler;
    }

    public static void reset() {
        OutputManager.outputHandler = new DefaultOutputHandler();
    }

    public static OutputHandler get() {
        if (outputHandler == null) reset();
        return outputHandler;
    }
}